### DISCUSSION TITLE
[Esame 3 giugno 2024] Esercizio 4

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#457] [Esami M-Z]
![3 giu es.4](https://github.com/user-attachments/assets/2b9e22f6-9b72-480b-9275-e0f716657896)

**Comment by thegeek-sys** [comment]
![immagine](https://github.com/user-attachments/assets/540f38ef-5ff4-4ca0-92fc-a8e010d73175)

**Comment by AglaiaNorza** [comment]
![image](https://github.com/user-attachments/assets/31d23ac9-7505-4a0d-a7d3-a48aec2aa5e4)

**Comment by niccolo-fato** [comment]
<img width="1280" height="791" alt="image" src="https://github.com/user-attachments/assets/9ef735c7-7ef0-47de-b1a1-1f1e45596b03" />
